scu-es/src/fiskaltrust.Middleware.SCU.ES.VeriFactu/Soap/Client.cs

```diff
@@ -25,44 +26,44 @@ public Client(HttpClient httpClient)
         _httpClient = httpClient;
     }
 
-    public async Task<Result<RespuestaRegFactuSistemaFacturacion, Error>> SendAsync(Envelope<RequestBody> envelope)
+    public async Task<(Result<RespuestaRegFactuSistemaFacturacion, Error> result, GovernmentAPI governmentAPI)> SendAsync(Envelope<RequestBody> envelope)
     {
         var requestString = envelope.XmlSerialize();
         var response = await _httpClient.PostAsync("", new StringContent(requestString, Encoding.UTF8, "application/soap+xml"));
 
+        var governmentAPI = new GovernmentAPI
+        {
+            Request = requestString,
+            Response = await response.Content.ReadAsStringAsync(),
+            Version = GovernmentAPISchemaVersion.V0
+        };
 
         if (!response.IsSuccessStatusCode || response.Content.Headers.ContentType?.MediaType == "text/html")
         {
-            return new Error.Http(response.StatusCode, await response.Content.ReadAsStringAsync());
+            return (new Error.Http(response.StatusCode, governmentAPI.Response), governmentAPI);
```

Comment:
Should we also include the Request here? For debug reasons it could help having it available